### PR TITLE
fix(block): extra null block request

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/753

### CODE REVIEW

File: src/pages/BlockDetail/index.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default () => {
   const queryBlockTransactions = useQuery(
     ['block-transactions', blockHash, currentPage, pageSizeParam, filter],
     async () => {
-      assert(blockHash != null)
+      assert(blockHash !== '')
```
Comment (Keith-CY) [inline src/pages/BlockDetail/index.tsx:82]: How about simply `!blockHash` to cover falsy values

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
https://ckb-explorer-frontend-in-magickbase-repo-git-a03497-magickbase.vercel.app/en/block/14139443

https://github.com/user-attachments/assets/aadd33c7-44ea-4843-9b94-0fdb40b3aea8

It was found that the interface 404 problem still exists, and there is no data in the page list.  @Daryl-L

**Comment by Keith-CY** [comment]
> [ckb-explorer-frontend-in-magickbase-repo-git-a03497-magickbase.vercel.app/en/block/14139443](https://ckb-explorer-frontend-in-magickbase-repo-git-a03497-magickbase.vercel.app/en/block/14139443)
> 
>  2024-08-05.11.12.00.mov 
> It was found that the interface 404 problem still exists, and there is no data in the page list. @Daryl-L

I can visit the page 
image

maybe just a temporary network issue in the staging environment.

**Comment by FrederLu** [comment]
> > [ckb-explorer-frontend-in-magickbase-repo-git-a03497-magickbase.vercel.app/en/block/14139443](https://ckb-explorer-frontend-in-magickbase-repo-git-a03497-magickbase.vercel.app/en/block/14139443)
> > 2024-08-05.11.12.00.mov
> > It was found that the interface 404 problem still exists, and there is no data in the page list. @Daryl-L
> 
> I can visit the page image
> 
> maybe just a temporary network issue in the staging environment.

Now there is no 404 problem. We will observe it after sending it to the Testnet environment.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**